reflex.conf.example

```diff
@@ -1,8 +1,8 @@
 # restart app server when a .go file changes
--s -r '\.go$' -- sh -c 'go run main.go server'
+-s -r '(\.go|manifest.json)$' -- sh -c 'go run main.go server'
```

Comment:
@verheyenkoen this is a better regex: `\.go$|^manifest.json$`, no need for capturing